submissions/task_html-css-popup/style.css

```diff
@@ -0,0 +1,388 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+/* DECLARE .HEADER */
+
+.header {
+    display: flex;
+    justify-content: flex-end;
+    width: calc(100%-80px);
+    height: 50px;
+    margin: 10px 10px;
+    padding: 2px 2px;
+    border: solid black 1px;
+    box-shadow: 1px 2px;
+    background-color:blue;
+}
+
+/* HEADER - Hide dots in the <ul> element*/
+
+.menu__list {
+    list-style: none;
+}
+
+/* HEADER - Style for <a> element */
+
+a {
+    text-decoration-style: dashed;
+    text-decoration-color: white;
+    text-underline-offset: 3px;
+}
+
+a:focus {
+    outline: 3px dashed gold;
+    text-decoration: underline;
+}
+
+a:hover,
+a:active {
+    text-decoration: underline;
+    text-decoration-color: white;
+}
+
+/* DECLARE .MAIN */
+
+.main {
+    display: flex;
+    width: calc(100%-80px);
+    height: 75vh;
+    margin: 10px 10px;
+    padding: 2px 2px;
+    border: solid black 1px;
+    box-shadow: 1px 2px;
+    align-items: center;
+    justify-content: center;
+
+}
+
+/* DECLARE .FOOTER */
+
+.footer {
+    display: flex;
+    width: calc(100%-80px);
+    height: 40px;
+    margin: 10px 10px;
+    padding: 2px 2px;
+    border: solid black 1px;
+    box-shadow: 1px 2px;
+    background-color:blue;
+}
+
+/* DECLARE WRAPPER */
+
+.wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    max-width: 100%;
+    min-width: 320px;
+    height: 100vh;
+    margin: 0 auto;
+}
+
+/* MENU + LIST-LINK-ITEM - BEM naming convention */
+
+.menu__burger-icon:focus {
+    outline: 3px dashed gold;
+    text-decoration: underline;
+}
+
+.menu__list {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.menu__link {
+    font-weight: 600;
+    font-size: 14px;
+    color: #0008;
+}
+
+.menu__item {
+    margin-right: 30px;
+}
+
+.menu__item:last-child {
+    margin-right: 0;
+}
+
+.menu__item:nth-child(3) {
+    position: relative;
+}
+
+.menu__user--icon img {
+    display: block;
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+}
+
+.menu__burger-checkbox {
+    position: absolute;
+    appearance: none;
+}
+
+.more-btn__checkbox {
+    position: absolute;
+    appearance: none;
+}
+
+.menu__burger-icon {
+    display: block;
+    position: relative;
+    width: 30px;
+    height: 30px;
+    background-image: url(../menu.png);
+    background-color: transparent;
+    background-position: center;
+    background-size: cover;
+}
+
+.menu__burger-icon:hover:before,
+.menu__burger-checkbox:checked+label .menu__burger-icon:before,
+.menu__burger-checkbox:focus+label .menu__burger-icon:before {

```

Comment:
Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
use class names instead

Reply:
Done.